libyul/AST.h

```diff
@@ -45,7 +45,7 @@ struct DebugData
 	}
 };
 
-struct TypedName { std::shared_ptr<DebugData const> debugData; YulString name; Type type; };
+struct TypedName { std::shared_ptr<DebugData const> debugData; YulString name{}; Type type{}; };
```

Comment:
I think making everything except the debug data optional is very dangerous.

Reply:
What do you suggest. Also adding `{}` to `debugData` or removing it again? (the latter would cause quite a few inconveniences in Parser.cpp. no blocker though)